requirements.txt

```diff
@@ -10,3 +10,4 @@ emcee>=3,<5
 tqdm
 joblib
 sacc~=0.2
```

Comment:
This should be bumped to 0.3 I think.

Reply:
done.  shall we add camb to this too?

Reply:
Nope. Only direct dependencies should be in this file. A dependence of a dependence need not be.